user_docs/en/changes.t2t

```diff
@@ -24,15 +24,11 @@ What's New in NVDA
 
 
 == Changes for Developers ==
+- The [NVDA API Announcement mailing list https://groups.google.com/a/nvaccess.org/g/nvda-api/about] was created. (#13999)
+-
 
 
 === Deprecations ===
-These are proposed API breaking changes.
-The deprecated part of the API will continue to be available until the specified release.
-If no release is specified, the plan for removal has not been determined.
-Note, the roadmap for removals is 'best effort' and may be subject to change.
-Please test the new API and provide feedback.
-For add-on authors, please open a GitHub issue if these changes stop the API from meeting your needs.

```

Comment:
I'm not sure about removing this. The initial intent was to make the policy and requirements as clear as possible purely from the change log. My guess is that this is much less likely to be read in the developer guide. The other issue is visibility of changes to the policy. For those who read the changes file only, they will miss updates to the policy.

One option might be to link to the section of the developer guide, and explicitly note changes to the policy here when they change.

Reply:
>One option might be to link to the section of the developer guide, and explicitly note changes to the policy here when they change.

I think this is a good idea.